notification plugin not working with cap project with cds =>8.0.0

notification plugin not working with cap project with cds =>8.0.0

I am getting the same error 

.[2024-09-30 16:12:04]  INFO executing the "npx cds build --production" command...
.....The cds build plugin 'notifications' is not compatible with this @sap/cds-dk version. Update the npm package '@cap-js/notifications/lib/build' to the latest version.
[2024-09-30 16:12:10] ERROR the "before-all"" build failed: could not execute the "npx cds build --production" command: exit status 1
make: *** [Makefile_20240930161130.mta:28: pre_build] Error 1
[2024-09-30 16:12:10] ERROR could not build the MTA project: could not execute the "make -f Makefile_20240930161130.mta p=cf mtar= strict=true mode=" command: exit status 2
Error: could not build the MTA project: could not execute the "make -f Makefile_20240930161130.mta p=cf mtar= strict=true mode=" command: exit status 2

cds -v output:

@cap-js/asyncapi: 1.0.2
@cap-js/cds-types: 0.6.5
@cap-js/notifications: 0.2.1
@cap-js/openapi: 1.0.5
@cap-js/sqlite: 1.7.3
@sap/cds: 8.2.3
@sap/cds-compiler: 5.2.0
@sap/cds-dk: 8.2.3
@sap/cds-dk (global): 8.0.2
@sap/cds-fiori: 1.2.7
@sap/cds-foss: 5.0.1
@sap/cds-mtxs: 2.1.0
@sap/eslint-plugin-cds: 3.0.5
Node.js: v18.18.0
home: C:\Node\test_import_alert\node_modules\@sap\cds
test_import_alert: 1.0.0

--- a/cds-plugin.js
+++ b/cds-plugin.js
@@ -1,9 +1,8 @@
 const cds = require("@sap/cds/lib");
-const NotificationsBuildPlugin = require("./lib/build");
 
 if (cds.cli.command === "build") {
   // register build plugin
-  cds.build?.register?.('notifications', NotificationsBuildPlugin);
+  cds.build?.register?.('notifications', require("./lib/build"));
 }
 
 else cds.once("served", async () => {

